src/lib/copy.ts

```diff
@@ -27,14 +32,14 @@ export const copyText = async (text: string): Promise<void> => {
     }
 };
 
-export const copy = (element: HTMLElement, text: string) => {
+export const copy = (element: HTMLElement, options: Options) => {
     async function click() {
```

Comment:
We should rename this function to `handle` or something similar instead of click